### PR TITLE
Regenerate yarn.lock from scratch

### PR DESCRIPTION
**Pull request by dapplion**
**Motivation**

Easiest way to mass-fix all dependency vulnerabilities

:warning: this can break many parts of the code, requires extensive testing

**Description**

I just run `rm yarn.lock && yarn`

### CODE REVIEW

**Comment by wemeetagain** [comment]
closed and reopened to unfreeze CI

File: packages/beacon-node/test/unit/network/peers/datastore.test.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe("Eth2PeerDataStore", () => {
     expect(dbDatastoreStub.batch.calledOnce).to.equal(true);
   });
 
-  it("should persist to db the oldest item after max", async () => {
+  it.only("should persist to db the oldest item after max", async () => {
```
Comment (dapplion) [inline packages/beacon-node/test/unit/network/peers/datastore.test.ts]: Why is this test skipped? Add a comment above explaining the error

Comment (wemeetagain) [reply packages/beacon-node/test/unit/network/peers/datastore.test.ts]: needs to be removed

**Comment by wemeetagain** [comment]
closing in favor of #4597

**Pull request closed without merging**